src/desktop/main.cc

```diff
@@ -116,6 +184,137 @@ MAIN {
   bool wantsVersion = false;
   bool wantsHelp = false;
 
+  auto bundleIdentifier = app.appData["meta_bundle_identifier"];
```

Comment:
it's already defined on line 503

Reply:
Can we make the value getters consistent, i.e. `userConfig["meta_bundle_identifier"]` vs `app.appData["meta_bundle_identifier"]`?